docs/software/plugins/canned-message.md

```diff
@@ -128,13 +128,13 @@ For setting up the plugin you will
 need to execute a sequence like this:
 
     meshtastic --set canned_message_plugin_allow_input_source "_any"
-    meshtastic --set canned_message_plugin_messages "I'm fine|I'm out|I'm back|Need helping hand|Help me with saw|I need an alpinist|I need ambulance|Keep Calm|On my way|I will be late|I'm already waiting|We have company|Beer is cold|Roger"
     meshtastic --set canned_message_plugin_send_bell False
     meshtastic --set canned_message_plugin_enabled True
 
+    meshtastic --set-canned-message "What am I doing?|I'm fine|Don't follow me|I'm out|I'm back|Need helping hand|Help me with saw|I need an alpinist|I need ambulance|Keep Calm|On my way|Need 5 mins|I will be late|I'm already waiting|I couldn't join|We have company|Beer is cold|Roger"
+
 :::note
-You can define up to 50 messages with a total length 200 bytes.
-(We are working on a solution to extend total length.)
+You can define up to 50 messages with a total length 800 bytes.
```

Comment:
this is a future TODO or feature, but I've thought about putting a text input box in this section that will calculate the bytes of your message using javascript. Maybe it's not worth it, but I thought it might be a nice touch and I want to say it out loud so someone knows about the idea.

Reply:
I believe 800 characters will be more than enough for canned messages. A character counter here might be cool, but overall does not really useful here.